### PR TITLE
🆙📅 Update API names 📛

### PR DESCRIPTION
**Pull request by nturinski**
Fixes https://github.com/microsoft/vscode-cosmosdb/issues/1237 

image

### CODE REVIEW

File: src/experiences.ts [hunk 1]
```diff
@@ -85,10 +85,10 @@ export function getExperienceQuickPick(api: API): IAzureQuickPickItem<Experience
 
 // Mongo is distinguished by having kind="MongoDB". All others have kind="GlobalDocumentDB"
 // Table and Gremlin are distinguished from SQL by their capabilities
-const SQLExperience: Experience = { api: API.DocumentDB, longName: "SQL", description: "(DocumentDB)", shortName: "SQL", kind: DBAccountKind.GlobalDocumentDB, tag: "Core (SQL)" };
-const MongoExperience: Experience = { api: API.MongoDB, longName: "MongoDB", shortName: "MongoDB", kind: DBAccountKind.MongoDB, tag: "Azure Cosmos DB for MongoDB API" };
-const TableExperience: Experience = { api: API.Table, longName: "Azure Table", shortName: "Table", kind: DBAccountKind.GlobalDocumentDB, capability: 'EnableTable', tag: "Azure Table" };
-const GremlinExperience: Experience = { api: API.Graph, longName: "Gremlin", description: "(Graph)", shortName: "Gremlin", kind: DBAccountKind.GlobalDocumentDB, capability: 'EnableGremlin', tag: "Gremlin (graph)" };
+const SQLExperience: Experience = { api: API.DocumentDB, longName: "Core", description: "(SQL)", shortName: "Core (SQL)", kind: DBAccountKind.GlobalDocumentDB, tag: "Core (SQL)" };
```
Comment (EricJizbaMSFT) [inline src/experiences.ts]: I'm confused - why is shortName longer than longName? I honestly don't remember where each of these is used...

Comment (nturinski) [reply src/experiences.ts]: I think it's only used here:
`const creatingMessage: string = `Creating Cosmos DB account "${wizardContext.accountName}" with API "${wizardContext.defaultExperience.shortName}"...`;`

And in the label for `AttachedAccountsTreeItem`. 
image

**Comment by nturinski** [comment]
Is it worth renaming the events too? I know it'd probably mess with dashboards?

File: src/experiences.ts (hunk 1 above)
Comment (EricJizbaMSFT) [reply src/experiences.ts]: Right so the shortName for core probably looks weird as an attached account due to nested parenthesis. I would think it should just be "Core"

**Comment by EricJizbaMSFT** [comment]
> Is it worth renaming the events too? I know it'd probably mess with dashboards?

User-facing strings are higher priority. No strong preference on changing the event names (but if you change them - I expect you to also update the dashboard and re-categorize all gdpr stuff)

**Comment by nturinski** [comment]
@fiveisprime Said he wants it to remain SQL 🤷‍♂

File: src/experiences.ts [hunk 2]
```diff
@@ -10,20 +10,20 @@ export enum API {
...
     Graph = 'Graph',
     Table = 'Table',
-    DocumentDB = 'DocumentDB'
+    Core = 'Core'
 }
 
 export enum DBAccountKind {
     MongoDB = 'MongoDB',
-    GlobalDocumentDB = 'GlobalDocumentDB'
+    GlobalCore = 'GlobalCore'
```
Comment (EricJizbaMSFT) [inline src/experiences.ts]: Kind is used when creating a cosmos db account here: https://github.com/microsoft/vscode-cosmosdb/blob/master/src/tree/CosmosDBAccountWizard/CosmosDBAccountCreateStep.ts#L24

Most of your changes are just for labels, but this one likely affects functionality. Are we sure Azure supports "GlobalCore" as the kind when creating an account?

**Pull request merged**